cmd/state-custom-installer/docker/centos/runBuild.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Please use the activestate.yaml for scripts (this goes for all .sh scripts introduced in this PR).

Note you can create a new `activestate.installer.yaml` or `activestate.<whatever>.yaml` if you want to keep it organized. It'll automatically get merged together at runtime.